hyperspy/component.py

```diff
@@ -119,8 +120,50 @@ def __init__(self):
         self.component = None
         self.grad = None
         self.name = ''
+        self.units = ''
```

Comment:
I think I will remove it from here. Now that I look at it, no idea why I added it in the first place.. I guess I thought that we had it to begin with